distribution/docs/src/main/resources/_contents/_tables/NSILI_Connected_Source-table-contents.adoc

```diff
@@ -0,0 +1,222 @@
+.[[NSILI_Federated_Source]]NSILI Federated Source
```

Comment:
It doesn't look like this NSILI Federated Source table should be in this file based on the file name.